arm-billing/2017-02-27-preview/swagger/billing.json

```diff
@@ -0,0 +1,399 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+                "$ref": "#/definitions/Invoice"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "description": "Lists all of the available billing REST API operations.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DownloadUrl": {
+      "description": "A secure URL that can be used to download a PDF invoice until the URL expires.",
+      "properties": {
+        "expiryTime": {
+          "description": "The time in UTC at which this download URL will expire.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "url": {
+            "description": "The URL to the PDF file.",
+            "type": "string",
+            "readOnly": true
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "The details of the error.",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string",
+          "readOnly": true
+        },
+        "target": {
+          "description": "The target of the particular error.",
+          "type": "string",
+          "readOnly": true
+        }
+      }      
+    },
+    "ErrorResponse": {
+      "description": "Error reponse indicates that the service is not able to process the incoming request. The reason is provided in the error message.",
+      "type": "object",
+      "properties": {
+        "error": {
+           "$ref": "#/definitions/InvoiceProperties"
+        }
+      }
+    },
+    "Invoice": {
+      "description": "An invoice resource can be used download a PDF version of an invoice.",
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
/cc @dsgouda - Please track this conversation. It may help us better understand the constraints and pattern of a tracked resource. (It would be interesting to see if location is not required).